refstis/functions.py

```diff
@@ -254,7 +254,7 @@ def get_start_and_endtimes(input_list):
     times.sort()
     start_mjd = times[0]
     end_mjd = times[-1]
-    times = Time(times, format = 'mjd', scale = 'utc', out_subfmt = 'date').iso
+    times = np.array([x.split()[0] for x in Time(times, format = 'mjd', scale = 'utc').iso])
```

Comment:
tested this in a notebook and it works perfectly. With the old line I found the same "date" error just from using just this line in a new environment. Great work finding this so quickly!

Reply:
Thanks, Matt!
I wanted to make sure there weren't any other dependency changes, so I ran the CTI code end-to-end to make sure.  We should implement better test coverage to catch these issues in both packages before it reaches the help desk.